packages/experimental/Divider/jest.config.js

```diff
@@ -0,0 +1,2 @@
+const { configureReactNativeJest } = require('@fluentui-react-native/scripts');
+module.exports = configureReactNativeJest('android');
```

Comment:
Should this be win32, given we are only showing the component on win32 to begin with?

Reply:
Yes it should. This is the initial jest test page created during bootstrapping of the component so this will be changed.